hooks/13_LoginForm/Readme.md

```diff
@@ -111,7 +111,13 @@ _./src/index.html_
 - Let's build a proper _login_ layout, _loginPage.tsx_. To build a nice layout, we will install _@material-ui/core_ and _@material-ui/icons_
 
 ```bash
-npm install @material-ui/core @material-ui/icons --save
+npm install @material-ui/core@^4.0.1 @material-ui/icons@^4.0.1 --save
```

Comment:
I think now they release it, if you make an npm install you will get the latest supporting hooks, am I right?

Reply:
Indeed, if they follow the guide, when launching npm install of the libraries, the latest version with support for hooks will be installed. My mistake